### PR TITLE
fix: native scene launch too slow

### PR DESCRIPTION
**Pull request by dogeFu**
Re: #https://github.com/cocos/3d-tasks/issues/14605

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by dumganhar — APPROVED** [review]

File: native/cocos/engine/Engine.cpp [hunk 1]
```diff
@@ -264,7 +264,7 @@ void Engine::tick() {
         ++_totalFrames;
 
         // iOS/macOS use its own fps limitation algorithm.
-#if (CC_PLATFORM == CC_PLATFORM_ANDROID || CC_PLATFORM == CC_PLATFORM_WINDOWS || CC_PLATFORM == CC_PLATFORM_OHOS) || (defined(CC_SERVER_MODE) && (CC_PLATFORM == CC_PLATFORM_MAC_OSX))
+#if (CC_PLATFORM == CC_PLATFORM_ANDROID || (CC_PLATFORM == CC_PLATFORM_WINDOWS && !defined(CC_EDITOR)) || CC_PLATFORM == CC_PLATFORM_OHOS) || (defined(CC_SERVER_MODE) && (CC_PLATFORM == CC_PLATFORM_MAC_OSX))
```
Comment (dumganhar) [inline native/cocos/engine/Engine.cpp:268]: Add a comment for why editor doesn't need the `sleep` logic.

File: native/cocos/renderer/gfx-gles3/GLES3GPUContext.cpp [hunk 2]
```diff
@@ -109,8 +109,11 @@ bool GLES3GPUContext::initialize(GLES3GPUStateCache *stateCache, GLES3GPUConstan
     }
 
     EGL_CHECK(eglBindAPI(EGL_OPENGL_ES_API));
-
+#if CC_EDITOR
```
Comment (hana-alice) [inline native/cocos/renderer/gfx-gles3/GLES3GPUContext.cpp]: any other choice? CC_EDITOR in engine make editor and engine glue with each other

File: native/cocos/engine/Engine.cpp (hunk 1 above)
Comment (dogeFu) [reply native/cocos/engine/Engine.cpp:268]: ok，comment was added.

File: native/cocos/renderer/gfx-gles3/GLES3GPUContext.cpp (hunk 2 above)
Comment (dogeFu) [reply native/cocos/renderer/gfx-gles3/GLES3GPUContext.cpp]: editor's code is already glue with engine，not just here,maybe later we can find a better way to do this.

**Review by dumganhar — APPROVED** [review]

**Pull request merged**